### PR TITLE
Add Ideas section

### PR DESCRIPTION
**Pull request by ZorianaPro**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -12,53 +16,55 @@
   </component>
   <component name="FileEditorManager">
     <leaf SIDE_TABS_SIZE_LIMIT_KEY="300">
-      <file pinned="false" current-in-tab="false">
+      <file pinned="false" current-in-tab="true">
```
Comment (yoanad) [inline .idea/workspace.xml:19]: What's this file for? Is it maybe committed by mistake?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -2,7 +2,11 @@
 <project version="4">
```
Comment (yoanad) [inline .idea/workspace.xml:2]: What's this file for? Is it maybe committed by mistake?

Comment (ZorianaPro) [reply .idea/workspace.xml:2]: I dont know. I dont update it just gets updated itself. I even ignored it in .gitignore.

Comment (yoanad) [reply .idea/workspace.xml:2]: Shouldn't it be `.idea/`?
I think you also have to remove the folder, commit, like they said here: https://stackoverflow.com/questions/32384473/gitignore-not-ignoring-idea-path 

Or just merge this and I can try to do it separately

**Pull request merged**